lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -700,11 +700,15 @@ def add_shopper_id(xml, options)
         if options[:ip] && options[:session_id]
           xml.session 'shopperIPAddress' => options[:ip], 'id' => options[:session_id]
```

Comment:
Hey Ruben, maybe we can include this new definition here:

```suggestion
          add_shopper_ip_address(xml, options)
          xml.session 'id' => options[:session_id]
```